sources/platform/console/two-factor-authentication.md

```diff
@@ -0,0 +1,97 @@
+
+### Confirmation
+
+When you close the setup process, you should see that your two-factor authentication is enabled in the account settings.
+
+![Apify console two-factor authentication section enabled](./images/console-account-two-factor-enabled.png)
+
+
+## Verification after sign-in
+
+After you enable two-factor authentication, the next time you attempt to sign in, you'll need to enter a code before you can get into the Apify Console. To do that, open your authenticator app and enter the code for your Apify account into the **code** field. After you enter the code, click on the **Verify** button and if the provided code is correct, you will proceed to Apify Console.
+
+![Apify console two-factor authentication form](./images/console-two-factor-authentication.png)
+
+## Using recovery codes
+
+In case you lose access to your authenticator app, you can use the recovery codes to sign in to your account. To do that, click on the **recovery code or begin 2FA account recovery** link below the **Verify** button. This will redirect you to a similar view as the current one, but instead of code from the authenticator app, you will need to enter one of the 16 recovery codes that you have received during the setup process.
+
+If the provided recovery code is correct, you will proceed to Apify Console same as if you provided the code from the authenticator app. After gaining access to Apify Console we recommend going to the [account settings](https://console.apify.com/account/settings), disabling the two-factor authentication there, and then enabling it again with the new authenticator app.
+
+:::info Removal of recovery codes
+When you successfully use a recovery code, we remove the code from the original list as it's no longer possible to use it again. If you use all of your recovery codes, you will not be able to sign in to your account with them anymore and you will need to either use your authenticator app or contact our support to help you regain access to your account.
+:::
+
+
+![Apify console two-factor authentication with recovery code form](./images/console-two-factor-use-recovery-code.png)
+
+## Disabling two-factor authentication
+
+In case you no longer want to use the two-factor authentication or if you lose access to your authenticator app, you can disable the two-factor authentication in your [account settings](https://console.apify.com/account/settings). To do that, scroll down to the **Two-factor authentication** section and click on the **Disable** button. We will ask you to enter either your verification code from the authenticator app or if you do not have access to it anymore, you can use one of your recovery codes. After you enter the code, click on the **Remove app** button which will verify the provided code and if it's valid, it will disable the two-factor authentication and remove the configuration from your account.
```

Comment:
```suggestion
If you no longer want to use the two-factor authentication or lose access to your authenticator app, you can disable the two-factor authentication in your [account settings](https://console.apify.com/account/settings). To do that, scroll down to the **Two-factor authentication** section and click on the **Disable** button. We will ask you to enter either your verification code from the authenticator app or, if you do not have access to it anymore, you can use one of your recovery codes. After entering the code, click on the **Remove app** button to verify the provided code. If it's valid, it will disable the two-factor authentication and remove the configuration from your account.
```